pkg/update/client_test.go

```diff
@@ -425,37 +385,12 @@ func TestCheckForUpdates_BehaviorOverTime(t *testing.T) {
 	}
 
 	// Finally we should check once more
-	clock.Advance(3 * time.Second)
+	clock.Advance(4 * time.Second)
 	repo.Reset()
 	_, _, _ = client.CheckForUpdates("my-cli", "v1.2.3", false)
 	req.True(repo.GetLatestMajorAndMinorVersionCalled())
 }
 
-func TestCheckForUpdates_NoCheckFileGiven(t *testing.T) {
```

Comment:
I removed this one because the test condition doesn't make sense anymore without a CheckFile.